scripts/noshake.json

```diff
@@ -57,6 +57,7 @@
   "Mathlib.Mathport.Rename",
   "Mathlib.Probability.Notation",
   "Mathlib.Tactic.Abel",
+  "Mathlib.Tactic.Algebraize",
```

Comment:
This is all well and good but there are now other files in mathlib with an unnecessary `import Mathlib.Tactic.Algebraize` because the file you added it to is imported by quite a lot of other files. Not sure if that should be a separate PR though? Up to you but ultimately I guess probably the unnecessary imports should be removed

Reply:
Will do in a separate PR.